test/testAdaptors/Vesting.t.sol

```diff
@@ -157,37 +129,35 @@ contract CellarVestingTest is Test {
         );
     }
 
-    function testFailWithdrawMoreThanVested() external {
+    function testFailWithdrawMoreThanVested(uint256 assets) external {
+        assets = bound(assets, 0.1e6, totalDeposit / 10);
         Cellar.AdaptorCall[] memory data = new Cellar.AdaptorCall[](1);
         bytes[] memory adaptorCalls = new bytes[](1);
 
-        // Deposit 5% of holdings, allowed under deviation
-        adaptorCalls[0] = _createBytesDataToDeposit(vesting, totalDeposit / 20);
+        // Deposit % of holdings, allowed under deviation
+        adaptorCalls[0] = _createBytesDataToDeposit(vesting, assets);
         data[0] = Cellar.AdaptorCall({ adaptor: address(vestingAdaptor), callData: adaptorCalls });
 
-        // Check event emission
-        vm.expectEmit(true, false, false, true);
```

Comment:
I don't think these are needed really, but let me know what you think.

Reply:
Yeah I am fine with not checking for event emits

Reply:
Perfect